provisioning/service/src/Twin/InitialTwinPropertyCollection.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
     /// Represents a collection of properties for device twin.
     /// </summary>
     [JsonConverter(typeof(InitialTwinPropertiesJsonConverter))]
-    public class InitialTwinPropertyCollection : IEnumerable
+    public class InitialTwinPropertyCollection : IEnumerable<KeyValuePair<string, object>>
```

Comment:
We have similar classes in the 2 IoT hub clients as well. Do they need this change as well?